tests/parsers/test_javascript_parser.py

```diff
@@ -39,7 +39,7 @@ def test_generate_file_results(self):
 
         results: Dict[str, FileResult] = self.parser.results
         self.assertTrue(results)
-        self.assertTrue(len(results) == 2)
+        self.assertTrue(len(results) == 3)
```

Comment:
It seems there's no equivalent for testing TS, despite we have a TS testdata. Is it expected?

Reply:
You're right, we're missing a test suite for TypeScript indeed. This will probably look similar to JavaScript, but it's still a good idea. Will add this to the todo list.